src/common/imageio_module.h

```diff
@@ -231,6 +231,8 @@ int dt_imageio_get_index_of_storage(dt_imageio_module_storage_t *storage);
 /* add a module into the known module list */
 void dt_imageio_insert_storage(dt_imageio_module_storage_t *storage);
 
+void dt_imageio_scale_splitup(double *x, double *y);
```

Comment:
either this should have the scale string as parameter (and not use the conf string) and/or a comment should be added to clarify the semantic as as-is it is really not clear what this is about.

Reply:
`The function from the string stored in the "plugins/lighttable/export/resizing_factor" parameter of the configuration file generates the x and y fraction's elements to calculate the scaling factor and improve the readability of the string itself displayed in the "scale" field of the export settings.`

Does this comment clarify anything?

Reply:
Yes, I still don't like this global routine using an hidden prefs. I would prefer having a more generic routine taking a string as parameter and returnin x & y.

Reply:
You did not change this? Can you explain why? I mean I'd like to understand if there is good arguments to keep it the way it is.